src/main/resources/com/hcl/appscan/jenkins/plugin/Messages.properties

```diff
@@ -41,13 +41,21 @@ warning.sca= AppScan on Cloud (ASoC) now performs SAST and SCA analysis as separ
 warning.include.sca.AppScan360= Only the SAST scan is being executed. SCA is only available with AppScan on Cloud and is not included in this job.
 error.invalid.format.include.sca = SCA scan requires an IRX file. Please provide the path to a valid IRX file to proceed.
 warning.sca.subscription = Only SAST scan execution takes place as SCA scan is not present under the active subscriptions.
-error.invalid.scan.id = This may occur due to incorrect characters in the Scan ID, a Scan ID from another organization, or insufficient access rights. Verify and try again.
-error.invalid.scan.id.application = The selected application does not match the application associated with the entered Scan ID. Verify and select the correct application.
-error.invalid.scan.id.scan.type = The selected scan type differs from the scan type of the entered Scan ID. Ensure the scan type matches the one associated with the Scan ID.
+error.invalid.scan.id = Invalid Scan ID: Ensure it belongs to your organization and matches the selected scan type.
+error.invalid.scan.id.application = Application mismatch: Ensure the selected application matches the Scan ID's associated application.
 error.invalid.scan.id.git.repo = Git Repository scans are not supported for SAST rescans via CI/CD.
-error.invalid.scan.id.ui = The entered Scan ID does not match the selected application and technology. Retrieve the correct ID from the AppScan on Cloud/AppScan 360� Server.
 warning.personal.scan.rescan = Personal Scan is not applicable for rescan.
 error.empty.scan.id = Scan ID value is empty. Verify and try again.
+warning.empty.execution.id = Base execution ID is missing. Since is scan is set to incremental, it will default to a full scan.
+error.target.wrong.input = Wrong user input. Please specify a directory to scan.
+error.scan.id.validation.rescan.allowed = Rescan not permitted: Demo scans or organization restrictions detected.
+error.scan.id.validation.status = The selected scan is in the {0} state. Rescans can only be performed when the scan is Ready, Paused, or Failed. Please wait or update the scan state.
+error.invalid.scan.id.ui = Invalid scan ID or mismatch in scan type.
+error.invalid.scan.id.application.ui = App mismatch with scan ID.
```

Comment:
This message should be more clear. I assume it's trying to say that the given scan Id does not exist in the selected application?